doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# Copyright (C) 2018 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+# This script is aim to setup or upgrade sos & uos automatically.
+
+# switch for download uos function
+skip_download_uos=0
+# turn on if you need to setup uos
+upgrade_uos=0
+
+function print_help()
+{
+echo "Usage:"
```

Comment:
Do you have a convention of indenting the body of a function?

Reply:
I just used tab in it...may I have any suggestion for improvement :smile:

Reply:
Should this line ``echo "Usage:"`` (and all the script lines in the body of this function) have a tab before them?  I don't see one.

Reply:
I got it. it will fix in next release.